vpu/generic/vpuGeneric/Dockerfile

```diff
@@ -0,0 +1,222 @@
+ARG BASE_VERSION=3
+
+##
+# Board architecture
+# arm or arm64
+##
+ARG IMAGE_ARCH=
+
+##
+# Board GPU vendor prefix
+##
+ARG GPU=
+
+##
+# SoM
+##
+#ARG MACHINE=verdin-imx8mm
+ARG MACHINE=verdin-imx8mp
+#ARG MACHINE=apalis-imx8
+
+# BUILD ------------------------------------------------------------------------
+FROM --platform=linux/${IMAGE_ARCH} \
+    torizon/wayland-base-vivante:${BASE_VERSION} AS Base
+
+ARG IMAGE_ARCH
+ARG GPU
+ARG MACHINE
+
+# Toradex unstable feed that provides NXP's downstream Gstreamer(v1.20) packages
+COPY toradex-debian-repo-unstable.gpg \
```

Comment:
Isn't this the same key as https://feeds.toradex.com/debian/toradex-debian-repo.gpg? Re-using https://github.com/toradex/torizon-containers/blob/rc/debian-docker-images/imx8/base-imx8/Dockerfile#L55 is cleaner, for example.

Also, I'd really advise against building a separate "unstable" feed instead of using a snapshot of the de-facto stable "testing" feed. We cannot rebuild this feed at any point in time, and this may bite us in the future. So something like `feeds.toradex.com/debian/staging-imx8` or something like that.

Reply:
Thanks for the review 🙏🏾 , I will reuse the key and move to a snapshot of `testing`.